extensions/wikia/AdEngine/tests/AdEngine2ContextServiceTest.php

```diff
@@ -45,84 +45,84 @@ private function mockMonetizationModule() {
 	public function adContextDataProvider() {
 		$defaultParameters = [
 			'titleMockType' => 'article',
-			'flags' => [],
-			'expectedOpts' => [],
+			'flags' => [ ],
+			'expectedOpts' => [ ],
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "expectedOpts" 15 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)